kiwi/bootloader/install/grub2.py

```diff
@@ -29,7 +29,7 @@
 
 class BootLoaderInstallGrub2(BootLoaderInstallBase):
     """
-        grub2 bootloader installation for x86 bios platform
+        grub2 bootloader installation
```

Comment:
correct and makes the purpose of the class clear and explicit like the code does too. But again you can also leave it as it is.